.npmrc

```diff
@@ -4,3 +4,4 @@ unsafe-perm = true
 frozen-lockfile = false
 legacy-peer-deps = true
 shamefully-hoist = true
+package-manager-strict=false
```

Comment:
Why is this needed?

Reply:
sorry, i forgot to remove that line, while project preparation i was stuck with version issue, i will remove that line soon